src/SourceGenerators/Uno.UI.SourceGenerators/XamlGenerator/XamlFileGenerator.cs

```diff
@@ -4714,10 +4723,16 @@ private string BuildBindingOption(XamlMemberDefinition m, INamedTypeSymbol? prop
 					return "null";
 				}
 
+				if (m.Member.Name == "TargetNullValue" && FindType(bindingType.Type) is INamedTypeSymbol namedTypeSymbol &&
+					m.Objects.SingleOrDefault()?.Members?.SingleOrDefault()?.Value is { } value)
+				{
+					return BuildLiteralValue(namedTypeSymbol, value.ToString());
+				}
```

Comment:
I'm not sure this logic is accurate/correct. But it at least fixes the linked issue. Can enhance if this is proven to not work in certain scenarios.